src/Library.php

```diff
@@ -69,7 +69,7 @@ public static function register_blocks() {
 		];
 		// Note: as a part of refactoring dynamic block registration, this will be moved
 		// to block level config.
-		if ( version_compare( $wp_version, '5.2', '>' ) ) {
+		if ( version_compare( $wp_version, '5.3', '>=' ) ) {
```

Comment:
I'm guessing you made the change here because it covers scenarios where the version might be `5.2.1` etc right? (in which case 💯 ).

Reply:
Yes, exactly!